featctl/.goreleaser.yml

```diff
@@ -0,0 +1,41 @@
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    ldflags:
+      - >
+        -s -w
+        -X github.com/oom-ai/oomstore/version.Version={{.Version}}
+        -X github.com/oom-ai/oomstore/version.Commit={{.Commit}}
+        -X github.com/oom-ai/oomstore/version.Built={{.CommitDate}}
+archives:
+  - replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
+    files:
+      - none*
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ incpatch .Version }}-next"
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - '^Merge'
+      - '^Update'
+      - '^docs[:(]'
+      - '^test[:(]'
+      - '^chore[:(]'
+      - '^refactor[:(]'
+      - '^ci[:(]'
```

Comment:
maybe need to add `feat`?

Reply:
This is a exlude list. `feat` commit messages should not be excluded because they compose the change log.